pina/model/layers/embedding.py

```diff
@@ -0,0 +1,142 @@
+""" Embedding modulus. """
+
+import torch
+from pina.utils import check_consistency
+
+
+class PBCEmbedding(torch.nn.Module):
```

Comment:
What about the extended class name? PeriodicBoundaryEmbedding maybe? Too many acronyms make things very complex for new user

Reply:
Yeah nice